app/services/hyrax/analytics/matomo.rb

```diff
@@ -154,6 +154,11 @@ def total_visitors(period = 'month', date = 'today')
           response["nb_visits_returning"].to_i + response["nb_visits_new"].to_i
         end
 
+        # TODO implement
```

Comment:
I didn't fix Matomo for this release, but I honestly think that this may be the only part that needs to be updated for Matomo 4 to work again. There is another task for adding Matomo 5